src/Compiler/FSComp.txt

```diff
@@ -1771,8 +1771,10 @@ featureReuseSameFieldsInStructUnions,"Share underlying fields in a [<Struct>] di
 3868,tcActivePatternArgsCountNotMatchOnlyPat,"This active pattern expects exactly one pattern argument, e.g., '%s pat'."
 3868,tcActivePatternArgsCountNotMatchArgs,"This active pattern expects %d expression argument(s), e.g., '%s%s'."
 3868,tcActivePatternArgsCountNotMatchArgsAndPat,"This active pattern expects %d expression argument(s) and a pattern argument, e.g., '%s%s pat'."
+featureAllowAccessModifiersToAutoPropertiesGettersAndSetters,"Allow access modifiers to auto properties getters and setters"
+3869,tcAccessModifiersNotAllowedInSRTPConstraint,"Access modifiers cannot be applied to an SRTP constraint."
 featureParsedHashDirectiveArgumentNonString,"# directives with non-quoted string arguments"
-3869,featureParsedHashDirectiveUnexpectedInteger,"Unexpected integer literal '%d'."
-3869,featureParsedHashDirectiveUnexpectedIdentifier,"Unexpected identifier '%s'."
+3870,featureParsedHashDirectiveUnexpectedInteger,"Unexpected integer literal '%d'."
+3870,featureParsedHashDirectiveUnexpectedIdentifier,"Unexpected identifier '%s'."
```

Comment:
Mhm this is probably okay but to be on the safe side I would not change warning numbers here. I know this happened because another features got introduced since this PR was created, sorry for inconvenience therefore.